packages/@ourworldindata/grapher/src/controls/Controls.scss

```diff
@@ -1,175 +1,408 @@
-$zindex-ControlsFooterMenu: 3;
-$zindex-ControlsFooter: 2;
+$light-stroke: #e7e7e7;
+$dark-stroke: #2d2d2d;
 
-.SelectEntitiesButton {
-    color: $controls-color;
+$light-fill: #dadada;
+$active-fill: #dbe5f0;
+$hover-fill: #f2f2f2;
 
-    svg {
-        margin-right: 3px;
-    }
-}
+$light-text: #858585;
+$dark-text: #5b5b5b;
+$active-text: #1d3d63;
 
-.ChangeEntityButton {
-    color: $controls-color;
-}
+$info-icon: #a1a1a1;
+$active-switch: #6e87a2;
 
-// FACET STRATEGY SELECTOR
-
-// the fake dropdown menu
-.FacetStrategyDropdown {
-    font-family: "Lato";
-    font-size: 12px;
-    font-weight: bold;
-    border: 1px solid #afb7c1;
-    border-radius: 5px;
-    padding-left: 10px;
-    padding-right: 10px;
-    cursor: pointer;
-}
+$medium: 400;
+$bold: 700;
+$lato: $sans-serif-font-stack;
```

Comment:
I copied these variables into `grapher.scss` when you sent them to me over Slack. I'm realising now that wasn't a good idea because these seem to be specific to the controls menu. I think I'll remove them from `grapher.scss` and refactor my css code. Does that make sense to you?

Reply:
Yeah, agreed. 

Keeping these sorts of variables local has been my approach both here and with the tooltips. There probably are a few things that could usefully be pulled up to the grapher level, but so far my preference has been to name them after their use-case within the component (rather than more generic names like blue-20, grey-80, etc.)